### PR TITLE
Newcomers_Guide.rst: Emphasize on low issues

### PR DESCRIPTION
**Pull request by harshit890**
Note is changed in order to emphasis more on
solving difficulty/low issue and start reviewing.

Closes coala#3578

### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.io/git#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://coala.io/commit#shortlog).
- [x] I have followed the [guidelines for the commit body](http://coala.io/commit#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://coala.io/commit#issue-reference).
- [x] I have [run all the tests](http://api.coala.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### CODE REVIEW

**Comment by harshit890** [comment]
@Makman2  sorry for closing https://github.com/coala/coala/pull/3579 ..closed it because i have commited some blunder mistake over there...so now u can review this and finally close this

**Comment by namanyadav12** [comment]
ack a439c46

**Comment by RohanVB** [comment]
Just a suggestion - why not have the text of difficulty/low and difficulty/newcomer as a link to the labelled page? as for newcomer would be : https://github.com/coala/coala/issues?q=is%3Aopen+is%3Aissue+label%3Adifficulty%2Fnewcomer

It'd make it easier for everyone in general.

**Comment by harshit890** [comment]
the link for the newcomer issue is already provided http://api.coala.io/en/latest/Developers/Newcomers_Guide.html#step-3-picking-up-an-issue in this

**Comment by harshit890** [comment]
better i m thinking to provide a link for the page where they can find PRs of pending review

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -338,14 +338,15 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
...
 
     However those who are familiar with opensource can start with
-    "difficulty/low" issues.
+    ``difficulty/low`` issues.
 
-    We highly encourage you to start reviewing other's "difficulty/low" or
-    newcomer issues after you complete your newcomer issue as reviewing helps,
-    you to learn more about coala and python.
+    We highly encourage you to start `reviewing <https://coala.io/review>`__
+    other's ``difficulty/low`` or``difficulty/newcomer`` issues after you
```
Comment (RohanVB) [inline docs/Developers/Newcomers_Guide.rst]: Run coala on your project to fix this before pushing :)

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -338,14 +338,15 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
...
+    then proceed to a ``difficulty/low`` issue.
 
     However those who are familiar with opensource can start with
-    "difficulty/low" issues.
+    ``difficulty/low`` issues.
 
-    We highly encourage you to start reviewing other's "difficulty/low" or
-    newcomer issues after you complete your newcomer issue as reviewing helps,
-    you to learn more about coala and python.
+    We highly encourage you to start `reviewing <https://coala.io/review>`__
```
Comment (RohanVB) [inline docs/Developers/Newcomers_Guide.rst]: This link part could go in a separate commit, since it doesn't do with the short log of the current commit.

Comment (harshit890) [inline docs/Developers/Newcomers_Guide.rst]: it would have been too much work than...making more commits and than squashing tham in to one

Comment (harshit890) [inline docs/Developers/Newcomers_Guide.rst]: my commit passes all test ..do u really want to make seperate commit for tht

Comment (namanyadav12) [inline docs/Developers/Newcomers_Guide.rst]: if you are going to add the link, then there need to two commits i think.
And you don't have to squash them into one, there will be two separate commits,
one for closing the issue and one  for adding the link.

Comment (RohanVB) [inline docs/Developers/Newcomers_Guide.rst]: > it would have been too much work than...making more commits and than squashing tham in to one

You wouldn't be squashing the commits into one, the two commits would be kept separate. 

> my commit passes all test ..do u really want to make seperate commit for tht

I know it seems a bit tedious, but it'll make work easier if particular changes are referenced for future contributors; not in this particular case since it's documentation - but for code in general, you should make changes specific to what the commit says it does :)

File: docs/Developers/Newcomers_Guide.rst [hunk 3]
```diff
@@ -338,14 +338,14 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: I think you should change commit message, it's the same as of the merged commit :+1:

File: docs/Developers/Newcomers_Guide.rst [hunk 4]
```diff
@@ -343,9 +343,10 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
     However those who are familiar with opensource can start with
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: Commit message:
```
Closes [xxx]
```
is wrong here, this commit doesn't close the issue ;)

File: docs/Developers/Newcomers_Guide.rst [hunk 5]
```diff
@@ -343,9 +343,10 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
...
     ``difficulty/low`` issues.
 
-    We highly encourage you to start reviewing other's ``difficulty/low`` or
-    ``difficulty/newcomer`` issues after you complete your newcomer issue, as
-    reviewing helps you to learn more about coala and python.
+    We highly encourage you to start `reviewing <https://coala.io/review>`__
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: is the double underscore right? or does it have to be a single underscore? I do not know RST that well...

Comment (harshit890) [inline docs/Developers/Newcomers_Guide.rst]: in the rst to link the file double underscore and single underscore both were used at different places...so i choose double underscore

File: docs/Developers/Newcomers_Guide.rst (hunk 4 above)
Comment (harshit890) [inline docs/Developers/Newcomers_Guide.rst]: the first or second commit

Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: `Newcomers_Guide.rst: Add link for review`

Comment (harshit890) [inline docs/Developers/Newcomers_Guide.rst]: so do i remove that line???

File: docs/Developers/Newcomers_Guide.rst [hunk 6]
```diff
@@ -343,9 +343,10 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
...
+    other's ``difficulty/low`` or ``difficulty/newcomer`` issues after you
+    complete your newcomer issue, as reviewing helps you to learn more about
```
Comment (Nosferatul) [inline docs/Developers/Newcomers_Guide.rst]: can we add that you help others too?

**Comment by harshit890** [comment]
Its already added in the starting note...

File: docs/Developers/Newcomers_Guide.rst [hunk 7]
```diff
@@ -343,9 +343,10 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
     However those who are familiar with opensource can start with
     ``difficulty/low`` issues.
 
-    We highly encourage you to start reviewing other's ``difficulty/low`` or
-    ``difficulty/newcomer`` issues after you complete your newcomer issue, as
-    reviewing helps you to learn more about coala and python.
+    We highly encourage you to start `reviewing <https://coala.io/review>`__
+    other's ``difficulty/low`` or ``difficulty/newcomer`` issues after you
```
Comment (adtac) [inline docs/Developers/Newcomers_Guide.rst]: But this doesn't give just difficulty/low or difficulty/newcomer issues. https://coala.io/review is the list of *all* PRs pending review. Either change the link or change this text.

**Comment by adtac** [comment]
ack https://github.com/coala/coala/pull/3580/commits/ac48c987df48f142c697135a6c1c099633f46c7c

**Comment by adtac** [comment]
unack https://github.com/coala/coala/pull/3580/commits/38019e467ceb87c021b25666f8c0f5f6d5b9ad02

File: docs/Developers/Newcomers_Guide.rst (hunk 7 above)
Comment (harshit890) [inline docs/Developers/Newcomers_Guide.rst]: there is no such link that provide such type of sorted list....atleast this link lead them to pending PRs or they will end up around wondering for tht.....and they will by themself review only those PRs in which they will find them comfortable........or else if u say i will remove tht link @adtac

Comment (adtac) [inline docs/Developers/Newcomers_Guide.rst]: No, rather remove the "``difficulty/low`` or ``difficulty/newcomer``" part - just say "other's issues".

**Comment by harshit890** [comment]
@adtac if u find everything ok now plzz merge it

**Comment by harshit890** [comment]
@Makman2

**Comment by Makman2** [comment]
commit message
>  Newcomers_Guide.rst: Emphasize on low issues
>
>Note is changed in order to emphasis more on
>solving difficulty/low issue and start reviewing.

doesn't really apply^^

**Comment by Makman2** [comment]
ack 94892ee

**Comment by Makman2** [comment]
unack 2d33670

**Comment by Makman2** [comment]
only commit message needs a bit love, then we merge ;)

**Comment by harshit890** [comment]
first line of note emphasize on solving low level issue

**Comment by Makman2** [comment]
Shortlog and body, both don't apply. (But don't remove the `Close xxx` reference, this one is correct! :D)

File: docs/Developers/Newcomers_Guide.rst [hunk 8]
```diff
@@ -338,13 +338,13 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
...
-    then proceed to a difficulty/low issues.
+    then proceed to a ``difficulty/low`` issue.
 
     However those who are familiar with opensource can start with
-    "difficulty/low" issues.
+    ``difficulty/low`` issues.
 
-    We highly encourage you to start reviewing other's "difficulty/low" or
-    newcomer issues after you complete your newcomer issue as reviewing helps,
+    We highly encourage you to start `reviewing <https://coala.io/review>`__
```
Comment (harshit890) [inline docs/Developers/Newcomers_Guide.rst]: @Makman2 and this line emphaisize on reviewing..

File: docs/Developers/Newcomers_Guide.rst [hunk 9]
```diff
@@ -338,13 +338,13 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
 
     **Do not only fix a newcomer issue!** It is highly recommended that you
     fix one newcomer issue to get familiar with the workflow at coala and
-    then proceed to a difficulty/low issues.
+    then proceed to a ``difficulty/low`` issue.
```
Comment (harshit890) [inline docs/Developers/Newcomers_Guide.rst]: @Makman2 this line emphasize on going for low level issues

**Comment by Makman2** [comment]
From gitter:
> @harshit890 a commit message should describe the action you perform in a commit
>the text you address emphasizes issue
>but what you do to the text is not emphasizing it more :grinning:
>you improve it
>(though "improve" is quite general in a commit message, the issue covers multiple stuff so having sth like Newcomers_Guide.rst: Improve would be okay for me)

**Comment by Makman2** [comment]
where's the commit gone with the link @harshit890 ? :O

File: docs/Developers/Newcomers_Guide.rst [hunk 10]
```diff
@@ -343,9 +343,9 @@ go for `a low issue <https://coala.io/low>`__ - they are really rewarding!
     However those who are familiar with opensource can start with
     ``difficulty/low`` issues.
 
-    We highly encourage you to start reviewing other's ``difficulty/low`` or
-    ``difficulty/newcomer`` issues after you complete your newcomer issue, as
-    reviewing helps you to learn more about coala and python.
+    We highly encourage you to start `reviewing <https://coala.io/low>`__
```
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: this is the wrong link, it refers to all low issues, not pending review PRs^^

**Comment by harshit890** [comment]
donw now sry for the last commit ...forgot to edit tht

File: docs/Developers/Newcomers_Guide.rst (hunk 10 above)
Comment (Makman2) [inline docs/Developers/Newcomers_Guide.rst]: also commit message:

Review link is add**ed** to the note to ease
newcomers in reaching the pending reviews
list.

**Comment by Makman2** [comment]
ack 19c2f60 b1ee5d1

**Comment by Makman2** [comment]
ack fe1b8f9

**Comment by harshit890** [comment]
@Makman2 if this PRs is approved i guess this should b merged

**Comment by Makman2** [comment]
Cherry-picked into https://github.com/coala/coala/pull/3594
